homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -125,18 +138,29 @@ def __init__(self, config, config_entry, discovery_hash):
 
     async def _subscribe_topics(self):
         """(Re)Subscribe to topics."""
+        value_template = self._config.get(CONF_VALUE_TEMPLATE)
+        if value_template is not None:
+            value_template.hass = self.hass
+        for tpl in self._templates.values():
```

Comment:
Please handle both templates in the same way, either store both in `self._templates` or both in local variables, and update the code accordingly.

Reply:
From looking at mqtt cover which has set_position_template as well as set_position_topic.  I have created separate local variables for each.